### DISCUSSION TITLE
Future of this library

### DISCUSSION THREAD

**Discussion by jclijmans** [cdmoro/bootstrap-vue-3#254] [General]
On the Bootstrap-Vue repo, I've seen mentioning a new start on the Vue3 implementation.
But, I've not seen a lot happening lately.

So my question is, what is the future of this library with regards to the official Vue3 implementation?
@cdmoro : Are you involved in this rewrite project? I've seen have been send an invitation to the bootstrap-vue org.

**Comment by aceofwings** [comment]
Wondering the same too.

**Reply by aceofwings** [reply]
@cdmoro

**Comment by cdmoro** [comment]
Hello there, sorry for the delay but I've been really busy those days. I'm currently moving abroad with my family because I get a new job, and with that comes a long list of things to do. On the other hand, as @jclijmans said, I have been invited to be part of the Bootstrap-vue developer team, this is something that I didn't want to miss. 

I would like to continue with this library, but I need some help here. So I'm open to adding maintainers who want to take care of the issues, PR, alongside.

Let me know what you think.
Thanks

**Reply by aceofwings** [reply]
Totally up to assist. Don't know if you have anything prior to onboarding additional maintainers.

**Reply by mariusa** [reply]
I can also help with issues, PR, some commits (like I did so far).

**Reply by foursmalltrees** [reply]
I would love to assist but it's my first time to contribute to OS projects
